py/desispec/night_qa.py

```diff
@@ -1911,24 +1919,63 @@ def write_nightqa_html(outfns, night, prod, css, surveys=None, nexp=None, ntile=
     html.write("\t<p>If a file appears in <span style='color:green;'>green</span>, it means it is present.</p>\n")
     html.write("\t<p>If a file appears in <span style='color:blue;'>blue</span>, it means it is a symlink to another file, the name of which is reported.</p>\n")
     html.write("\t<p>If a file appears in <span style='color:red;'>red</span>, it means it is missing.</p>\n")
-    html.write("\t<p>If a file appears in <span style='color:orange;'>orange</span>, it means it does not date from the corresponding night (likely done in the morning, in which case the pipeline uses some default files, if no special action has been taken by the data team).</p>\n")
+    html.write("\t<p>If a file appears in <span style='color:orange;'>orange</span>, it means it has been generated after the earliest file of the processed science exposures (likely done in the morning, in which case the pipeline uses some default files, if no special action has been taken by the data team).</p>\n")
     html.write("\t</br>\n")
     html.write("<table>\n")
+    # AR science exposure earliest timestamps per campet
+    earliest_sci_fns, earliest_sci_m_times = {}, {}
+    log.info("earliest science exposure timestamp per campet:")
+    for petal in petals:
+        for camera in cameras:
+            campet = "{}{}".format(camera, petal)
+            # AR list all science exposure files for that campet
+            fns = []
+            for expid in expids:
+                fn = findfile("frame", night, expid=expid, camera=camera+str(petal), specprod_dir=prod)
+                expdir = os.path.dirname(fn)
+                fns += sorted(glob(os.path.join(expdir, "*-{}-*".format(campet))))
```

Comment:
even though `os.path.getmtime` is fairly fast, metadata queries are historically a weak point of CFS, so I suggest that we limit this glob to reduce the number of files that will be inspected, e.g. to `"frame-{}-*.format(campet)"`.